lib/apruve/resources/corporate_account.rb

```diff
@@ -8,6 +8,7 @@ def self.find(merchant_id, email=nil)
       if email.nil?
         return find_all(merchant_id)
       end
+      email = URI::encode(email)
       response = Apruve.get("merchants/#{merchant_id}/corporate_accounts?email=#{email}")
```

Comment:
for the test, `email: 'foo+test@example.com'`, This string evaluates to `merchants/89ea2488fe0a5c7bb38aa7f9b088874b/corporate_accounts?email=foo+test@example.com`